### PR TITLE
docs(change log): add change log for 4.3.0

### PR DESCRIPTION
**Pull request by tigercl**

### CODE REVIEW

File: en_US/changes/changes-4.3.md [hunk 1]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+
+EMQ X 4.3.0 is released now, it mainly includes the following changes:
+
+### Features and Enhancement
+
+#### Building
+
+- Support Erlang/OTP 23
+- The new installation package only supports macOS 10.14 and above
+- Project adjusted to umbrella structure
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md:30]: - Support using Elixir to build plugins

File: en_US/changes/changes-4.3.md [hunk 2]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Support routing table compression, reduce memory usage, enhance subscription performance, publishing performance will be slightly affected, so disable option is provided
+- Improve wildcard subscription performance
+- Improve processing performance when a large number of clients are offline
+
+#### Safety
+
+- Protect EMQ X Broker from cross-site WebSocket hijacking attacks
+- SSL supports `verify` and `server_name_indication` configuration
+- Support the configuration of the maximum length of the certificate chain and the password of the private key file
+
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md:47]: Use tlsv1.3 by default, tlsv1.3 configs has no effect if started on OTP 22

File: en_US/changes/changes-4.3.md [hunk 3]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+#### Other
+
+- Added update resource logic for rule engine
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- Added update resource functionality for rule engine
```

File: en_US/changes/changes-4.3.md [hunk 4]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Rule engine SQL function supports conversion between unix timestamp and rfc3339 format time
+- Keep retrying the resources that failed to connect after the EMQ X Broker is started
+- Websocket listener supports selecting supported subprotocols from the subprotocols list
+- WebSocket connection supports obtaining real IP and Port
+- Support the default authentication method caching_sha2_password of MySQL 8.0
+- JWT authentication supports JWKS
+- The starting point is randomly selected when the shared subscription distribution strategy is configured as `round_robin`
+- Shared subscription supports Hash distribution of messages by source topic
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- Shared subscription supports hash distribution of messages by source topic
```

File: en_US/changes/changes-4.3.md [hunk 5]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Support import and export of Authentication & ACL information in Mnesia
+- Allow to use Base64 encoded client certificate or MD5 value of client certificate as username or Client ID
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- Allow to use base64 encoded client certificate or MD5 value of client certificate as username or Client ID
```

File: en_US/changes/changes-4.3.md [hunk 6]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Support restarting the listener
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- MQTT listener restart from API/CLI
```

File: en_US/changes/changes-4.3.md [hunk 7]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Only enable data telemetry in the official version
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]: delete this line.

File: en_US/changes/changes-4.3.md [hunk 8]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Support clearing all ACL cache
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- API/CLI to force evict ACL cache
```

File: en_US/changes/changes-4.3.md [hunk 9]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Support observer_cli
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
- Added observer_cli
```

File: en_US/changes/changes-4.3.md [hunk 10]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Quickly kill the connection process when OOM
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]: Fixed delayed connection process OOM kill

File: en_US/changes/changes-4.3.md [hunk 11]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Project adjusted to umbrella structure
+
+#### Performance improvement
+
+- The underlying implementation of the multi-language extension function is changed from erlport to gRPC
+- Support routing table compression, reduce memory usage, enhance subscription performance, publishing performance will be slightly affected, so disable option is provided
+- Improve wildcard subscription performance
+- Improve processing performance when a large number of clients are offline
+
+#### Safety
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]: Security

File: en_US/changes/changes-4.3.md [hunk 12]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Allow to use Base64 encoded client certificate or MD5 value of client certificate as username or Client ID
+- Support restarting the listener
+- Only enable data telemetry in the official version
+- Support clearing all ACL cache
+- Support observer_cli
+- Quickly kill the connection process when OOM
+- Support cluster metrics for Prometheus
+- Redis sentinel mode supports SSL connection
+- Support single-line log output, and support rfc3339 time format
+
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md:72]: please add:  ipv6 auto-probing

Comment (zmstone) [inline en_US/changes/changes-4.3.md:72]: Please add: JSON log format

File: en_US/changes/changes-4.3.md [hunk 13]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+#### Other
+
+- Added update resource logic for rule engine
+- Rule engine SQL function supports conversion between unix timestamp and rfc3339 format time
+- Keep retrying the resources that failed to connect after the EMQ X Broker is started
+- Websocket listener supports selecting supported subprotocols from the subprotocols list
+- WebSocket connection supports obtaining real IP and Port
+- Support the default authentication method caching_sha2_password of MySQL 8.0
+- JWT authentication supports JWKS
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]: move up to  "Security" section

File: en_US/changes/changes-4.3.md [hunk 14]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Fix the issue that the sleep mode of MQTT-SN is unavailable
+- Fix the issue that the MQTT-SN gateway will discard DISCONNECT packets in sleep mode
+- Fix the issue that the LwM2M gateway encodes and decodes numbers into unsigned integers
+
+#### Resource
+
+- Fix the issue that the MySQL authentication SSL/TLS connection function is not available
+- Fix Redis reconnection failure
+
+#### Other
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md]:
```suggestion
#### Other Fixes
```
Then the items do not have to all start with "Fix the issue"

File: en_US/changes/changes-4.3.md [hunk 15]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
 # Changes
 
+## 4.3.0 版本
+
+*Release Date: 2021-05-06*
+
+EMQ X 4.3.0 is released now, it mainly includes the following changes:
+
+### Features and Enhancement
+
```
Comment (zmstone) [inline en_US/changes/changes-4.3.md:25]: Add 2 non-compatible changes:

1.  emqx_auth_client and emqx_auth_username are merged to emqx_auth_mnesia
Please refer to doc (TODO: link) to export data from older versions, and import to v4.3

2. By default, docker only logs to console, set EMQX_LOG__TO=file to switch to file

File: en_US/changes/changes-4.3.md (hunk 12 above)
Comment (zmstone) [inline en_US/changes/changes-4.3.md:72]: Please add: environment variable override for configuration is made available for all distributions (previously only for docker)

Comment (zmstone) [inline en_US/changes/changes-4.3.md:72]: Please add : Certificate upload from dashboard has been made available for open-source edition (previously only for enterprise edition)

File: en_US/changes/changes-4.3.md [hunk 16]
```diff
@@ -15,6 +15,91 @@ ref: undefined
...
+- Fix the issue of inflight in MQTT bridge
+- Fixed the error of indicator statistics in the MQTT bridge function and the problem of multiple unit conversions in the `retry_interval` field
+- Fix the issue of incorrect calculation of alarm duration
+- Fix the issue that the long Client ID cannot be tracked
+- Fix the issue that the query client information may crash
+- Fix the issue of the inconsistency between topic rewriting and ACL execution order when publishing and subscribing
+- Fix the issue that the WebSocket connection cannot use the peer certificate as the username
+- Fix the issue that the authentication data cannot be imported
+- Fix the issue that EMQ X may fail to start in Docker
+
```
Comment (turtleDeng) [inline en_US/changes/changes-4.3.md:110]: Fix the issue MQTT-SN disconnects abnormally when clean_session = false and does not publish a will message

**Review by turtleDeng — APPROVED** [review]

**Pull request merged**